cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -141,7 +142,7 @@ users:
 			name: "kubeconfig flag",
 			options: &Options{
 				ComponentConfig: componentconfig.KubeSchedulerConfiguration{
-					ClientConnection: componentconfig.ClientConnectionConfiguration{
+					ClientConnection: apimachineryconfig.ClientConnectionConfiguration{
```

Comment:
I noticed this test isn't actually inspecting any values of the resulting config, just testing the user credentials.

I expanded it to look at the config that is loaded in https://github.com/kubernetes/kubernetes/pull/67352

That test will help us notice of any defaults change unexpectedly while moving things around, switching to shared structs, etc.